src/Microsoft.CSharp/src/Microsoft/CSharp/RuntimeBinder/CSharpConvertBinder.cs

```diff
@@ -121,7 +122,13 @@ public override DynamicMetaObject FallbackConvert(DynamicMetaObject target, Dyna
             {
                 return com;
             }
+#else
```

Comment:
In what situation is ENABLECOMBINDER set?  Is code under it just dead code that should be removed?

Reply:
Currently `ENABLECOMBINDER` is never set AFAIK. However, if we decide to bring back the COM binder for `dynamic` in the .NET 5 time frame (see https://github.com/dotnet/coreclr/issues/24246) then we would define it for .NET Core 5.

Reply:
Eeeh, probably not relevant for the issue, but what will be the difference between .NET 5 and .Net Core 5?

Reply:
.NET 5 and .NET Core 5 are the same thing. I still sometimes accidentally use the ".NET Core" branding when referring to .NET 5 by force of habit. Sorry for the confusion.